docs/resources/table_constraint.md

```diff
@@ -125,12 +125,15 @@ resource "snowflake_table_constraint" "unique" {
 <a id="nestedblock--foreign_key_properties"></a>
 ### Nested Schema for `foreign_key_properties`
 
+Required:
+
+- `references` (Block List, Min: 1, Max: 1) The table and columns that the foreign key references. Not applicable for primary/unique keys (see [below for nested schema](#nestedblock--foreign_key_properties--references))
```

Comment:
can we also adjust the description then? If this is nested in foreign key properties, than it does not have to be explicitly not applicable for primary/unique

Reply:
👍 adjusted